Dockerfile.dev

```diff
@@ -12,7 +12,9 @@ WORKDIR /usr/src/app
 
 RUN gem install bundler
 COPY Gemfile Gemfile.lock ./
-RUN bundle install
+RUN bundle install --full-index
 
-ENTRYPOINT cd /usr/src/app && hivemind -p 3021 ./Procfile
+ENTRYPOINT cd /usr/src/app && \
```

Comment:
Is the `cde /usr/src/app` needed? I thought the `WORKDIR` instruction also set the current working directory (see [the docs](https://docs.docker.com/engine/reference/builder/#workdir)).

Reply:
I've removed it everywhere. Thanks.